mysos/scheduler/scheduler.py

```diff
@@ -425,6 +469,18 @@ def statusUpdate(self, driver, status):
         log.error("Status update failed due to launcher error: %s" % e.message)
         self._stop()
 
+      # Update metrics.
+      # TODO(xujyan): This doesn't rule out duplicates, etc. We can consider updating these metrics
+      # in the launcher.
+      if status.state == mesos_pb2.TASK_FINISHED:
```

Comment:
instead of tasks can we call them updates instead to more accurately capture what is happening here?